VAMobile/src/screens/HealthScreen/MedicalRecordsScreen.tsx

```diff
@@ -22,12 +23,22 @@ const MedicalRecordsScreen = ({ navigation }: MedicalRecordsScreenProps) => {
   const navigateTo = useRouteNavigation()
   const { gutter } = theme.dimensions
 
+  const { WG_LabsAndTestsEnabled } = getWaygateToggles()
+  const isLabsAndTestsEnabled = WG_LabsAndTestsEnabled?.enabled
+
   return (
     <FeatureLandingTemplate
       backLabel={t('health.title')}
       backLabelOnPress={navigation.goBack}
       title={t('vaMedicalRecords.title')}>
       <Box mb={theme.dimensions.standardMarginBetween}>
+        {isLabsAndTestsEnabled && (
```

Comment:
This is implemented differently than the `Allergies` feature toggle. What is the preferred way? We should discuss release strategy.

Reply:
For this you likely want to use a feature toggle. The waygates provide more functionality around displaying errors or giving users instructions, but since this is a new feature we just want it to be there or not be there based on the config. Was there anything around release you wanted to discuss?